### PR TITLE
Userstory/meterui messages

### PR DESCRIPTION
**Pull request by fM4s0n**

### CODE REVIEW

File: Electrify.Dlms/Server/DlmsServer.cs [hunk 1]
```diff
@@ -37,6 +40,8 @@ public void AddObject(GXDLMSObject dlmsObject, AccessMode3 valueAccessMode = Acc
     
     public void SetEnergy(int energyValue)
     {
+        string blah = "";
```
Comment (hs2213) [inline Electrify.Dlms/Server/DlmsServer.cs:43]: remove?

Comment (fM4s0n) [reply Electrify.Dlms/Server/DlmsServer.cs:43]: lol

File: Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs [hunk 2]
```diff
@@ -68,6 +75,41 @@ private void UpdateDial()
...
+
+        static void toastOptions(ToastSettings options)
+        {
+            options.DisableTimeout = true;
+            options.ShowCloseButton = false;
+        }
+
+        if (connected)
+        {
+            ToastService.ShowError(_disconnectMessage, toastOptions);
```
Comment (hs2213) [inline Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs:91]: idk if its wrong or im misunderstanding, but why would you want to show the error message if its connected?

**Review by JoshCartledge3 — DISMISSED** [review]

File: Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs (hunk 2 above)
Comment (fM4s0n) [reply Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs:91]: because im dumb

Comment (fM4s0n) [reply Electrify.SmartMeterUi/Components/Pages/SmartMeterHome.razor.cs:91]: fixed

**Review by hs2213 — APPROVED** [review]

**Review by RoyalScribblz — APPROVED** [review]

**Pull request merged**